### PR TITLE
Add prefix for dcf manifest

### PR DESCRIPTION
**Pull request by vshand11**
implemented the logic to check if the id contains the prefix, if not add the prefix.

### CODE REVIEW

File: src/dcf_manifest_generator.py [hunk 1]
```diff
@@ -39,9 +39,11 @@ def generate_dcf(self):
         acl ="['*']" if not self.submission.get("controlledAccess", False) else f"['{self.submission.get('dbGaPID')}']"
         authz = "['/open']" if not self.submission.get("controlledAccess", False) else f"['/programs/{self.submission.get('dbGaPID')}']"
         url =  f's3://{self.config[PROD_BUCKET_CONFIG_NAME]}/{self.submission[DATA_COMMON_NAME]}/{self.submission.get("studyID")}/'
+        
         for r in file_nodes:
+            node_id = r[NODE_ID] if r[NODE_ID].startswith("dg.4DFC/") else "dg.4DFC/" + r[NODE_ID],
```
Comment (n2iw) [inline src/dcf_manifest_generator.py:44]: Could you extract the prefix into a constant? Thanks!

**Review by n2iw — CHANGES_REQUESTED** [review]

File: src/dcf_manifest_generator.py (hunk 1 above)
Comment (vshand11) [reply src/dcf_manifest_generator.py:44]: I convert lots string values to constants in new branch for CRDCDH-1355 and will be ready for your review soon.

Comment (vshand11) [reply src/dcf_manifest_generator.py]: updated.

**Review by n2iw — APPROVED** [review]
LGTM

**Pull request merged**